ui/admin/app/components/form/host-set/aws/index.hbs

```diff
@@ -236,6 +236,25 @@
     </:body>
   </Rose::Form::Fieldset>
 
+  <form.input
```

Comment:
This code looks the same used in `azure` template. Is that right?

If so: Should we move this to a component ? So we do not duplicate code.

Thoughts?

Reply:
I think that would make sense if it were an `app/template/azure`. This is a form component. I'm not quite sure how that would work here.

Reply:
I was meaning make the field as component. It is ok for now, in the future we might check if this makes sense and how might be best way to do it with Randy.